hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/RunCompactionActionExecutor.java

```diff
@@ -125,6 +125,7 @@ public HoodieWriteMetadata<HoodieData<WriteStatus>> execute() {
       compactionMetadata.setWriteStatuses(statuses);
       compactionMetadata.setCommitted(false);
       compactionMetadata.setCommitMetadata(Option.of(metadata));
+      compactionMetadata.setWriteStats(updateStatusMap);
```

Comment:
Does line 116 already set up the write stat?

Reply:
> Does line 116 already set up the write stat?

Yes, same stats value, but line 166 set up it in HoodieCommitMetadata but not in HoodieWriteMetadata. I think we should also set it in HoodieWriteMetadata.